.travis.yml

```diff
@@ -19,6 +19,7 @@ install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
+  - git clone https://github.com/OCA/stock-logistics-workflow -b ${VERSION} $HOME/stock-logistics-workflow
```

Comment:
could you add the following line to a new text file at the root of the repository called oca-dependencies.txt:

    stock-logistics-workflow